CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Castle Core Changelog
 
+## Unreleased
+
+Bugfixes:
+- Make ProxyUtil.IsAccessible(MethodBase) take into account declaring type's accessibility so it doesn't report false negatives for e.g. public methods in inaccessible classes. (@stakx, #289)
+
```

Comment:
1. I wouldn't, if anyone was inheriting this class, then bad luck 😉 
2. It could be breaking if someone re-compiles and has warnings as errors, however I think we just mention it when we actually remove them so we don't bloat the changelog.

> If so, under what heading do you usually list non-breaking changes?

I think I've used "Enhancements" in one of the changelogs but that isn't applicable here obviously. Something to decide when it comes up.